cmd/server/main.go

```diff
@@ -272,6 +275,13 @@ func (o *ProxyRunOptions) Validate() error {
 		}
 	}
 
+	switch o.proxyStrategy {
+	case "destIP":
+	case "default":
```

Comment:
It would be good if we started making our flags backward compatible. Specifically older scripts/containers should get the origianl/default behavior id the flag is missing. Can we make empty string ("") equivalent to default?